stubs/resources/views/flux/modal/close.blade.php

```diff
@@ -1,5 +1,7 @@
+@props(['class' => ''])
+
 @blaze
 
-<ui-close data-flux-modal-close>
+<ui-close data-flux-modal-close {{ $attributes->merge(['class' => $class]) }}>
```

Comment:
This can just be `{{ $attributes }}` as class will be forwarded automatically.

```suggestion
<ui-close data-flux-modal-close {{ $attributes }}>
```

Reply:
@joshhanley  Absolutely cleaner ! Thanks :)
Users can implement defaults as they need it...